package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -620,22 +649,44 @@ def _read_next_timestep(self):
             except IndexError:
                 raise ValueError("No coordinate information detected")
         elif not self.lammps_coordinate_convention in convention_to_col_ix:
-            raise ValueError(f"No coordinates following convention {self.lammps_coordinate_convention} found in timestep")
+            raise ValueError(f"No coordinates following convention "
+                "{self.lammps_coordinate_convention} found in timestep")
```

Comment:
PEP8 indentation needs to match the line above 

```suggestion
                                      "{self.lammps_coordinate_convention} found in timestep")
```